cli/test/playwright.integration.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const path = require('path');
+const { assert } = require('@sinonjs/referee-sinon');
+const execa = require('execa');
+
+describe('playwright', () => {
+  it('passes', async () => {
+    const result = await execa(
+      '../../index.js',
+      [
+        '--driver',
+        'playwright',
+        '--driver-option.engine',
+        'chromium',
```

Comment:
Did Firefox do strange things in CI or is this just for speed?

Reply:
There is no Firefox in the GitHub action container installed. However, chromium is. We'd need to figure out how to install Firefox, but then I thought it's not important for the purpose of this test.

Reply:
I was under the impression that Playwright installs all supported engines on `postinstall` so I am surprised this does not work. It does however place the executable outside of `node_modules` (as opposed to puppeteer), so maybe this needs some npm caching and/or $PATH adjustments in our context. I can look at that myself if you don't feel like doing it here.

See:
- https://playwright.dev/docs/ci#github-actions
- https://playwright.dev/docs/ci#caching-browsers

Reply:
I might have changed too many things at once when I tried to get the build working.

Reply:
I gave this another try and integration tests time out when I configure firefox as the engine. I'll merge this now and we can look into it separately. Once we have a solution locally and for GitHub Actions, we can document the setup in `driver-playwright/README.md`.